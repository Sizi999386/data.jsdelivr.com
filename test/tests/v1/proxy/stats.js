const { makeEndpointTests, setupSnapshots } = require('../../../utils');

const periodOptions = [ 'day', 'week', 'month', 'year', 'all', undefined ];

describe('/v1/proxy/stats', () => {
	before(() => {
		setupSnapshots(__filename);
	});

	makeProxyStatsTests();
});

function makeProxyStatsTests () {
	let defaults = {
		period: 'month',
	};

	makeEndpointTests('/v1/proxy/{name}/stats{?period}', defaults, [
		{ name: 'wp-plugins', period: periodOptions },
	]);
}
