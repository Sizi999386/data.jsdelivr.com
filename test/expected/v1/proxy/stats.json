{
	"/v1/proxy/wp-plugins/stats?period=all": {
		"date": "2022-04-21",
		"hits": {
			"total": {
				"total": 2346
			},
			"dates": {
				"2022-02-10": 0,
				"2022-02-11": 1,
				"2022-02-12": 2,
				"2022-02-13": 3,
				"2022-02-14": 4,
				"2022-02-15": 5,
				"2022-02-16": 6,
				"2022-02-17": 7,
				"2022-02-18": 8,
				"2022-02-19": 9,
				"2022-02-20": 10,
				"2022-02-21": 11,
				"2022-02-22": 12,
				"2022-02-23": 13,
				"2022-02-24": 14,
				"2022-02-25": 15,
				"2022-02-26": 16,
				"2022-02-27": 17,
				"2022-02-28": 18,
				"2022-03-01": 19,
				"2022-03-02": 20,
				"2022-03-03": 21,
				"2022-03-04": 22,
				"2022-03-05": 23,
				"2022-03-06": 24,
				"2022-03-07": 25,
				"2022-03-08": 26,
				"2022-03-09": 27,
				"2022-03-10": 28,
				"2022-03-11": 29,
				"2022-03-12": 30,
				"2022-03-13": 31,
				"2022-03-14": 32,
				"2022-03-15": 33,
				"2022-03-16": 34,
				"2022-03-17": 35,
				"2022-03-18": 36,
				"2022-03-19": 37,
				"2022-03-20": 38,
				"2022-03-21": 39,
				"2022-03-22": 40,
				"2022-03-23": 41,
				"2022-03-24": 42,
				"2022-03-25": 43,
				"2022-03-26": 44,
				"2022-03-27": 45,
				"2022-03-28": 46,
				"2022-03-29": 47,
				"2022-03-30": 48,
				"2022-03-31": 49,
				"2022-04-01": 50,
				"2022-04-02": 51,
				"2022-04-03": 52,
				"2022-04-04": 53,
				"2022-04-05": 54,
				"2022-04-06": 55,
				"2022-04-07": 56,
				"2022-04-08": 57,
				"2022-04-09": 58,
				"2022-04-10": 59,
				"2022-04-11": 60,
				"2022-04-12": 61,
				"2022-04-13": 62,
				"2022-04-14": 63,
				"2022-04-15": 64,
				"2022-04-16": 65,
				"2022-04-17": 66,
				"2022-04-18": 67,
				"2022-04-19": 68
			}
		},
		"bandwidth": {
			"total": {
				"total": 38474400
			},
			"dates": {
				"2022-02-10": 0,
				"2022-02-11": 16400,
				"2022-02-12": 32800,
				"2022-02-13": 49200,
				"2022-02-14": 65600,
				"2022-02-15": 82000,
				"2022-02-16": 98400,
				"2022-02-17": 114800,
				"2022-02-18": 131200,
				"2022-02-19": 147600,
				"2022-02-20": 164000,
				"2022-02-21": 180400,
				"2022-02-22": 196800,
				"2022-02-23": 213200,
				"2022-02-24": 229600,
				"2022-02-25": 246000,
				"2022-02-26": 262400,
				"2022-02-27": 278800,
				"2022-02-28": 295200,
				"2022-03-01": 311600,
				"2022-03-02": 328000,
				"2022-03-03": 344400,
				"2022-03-04": 360800,
				"2022-03-05": 377200,
				"2022-03-06": 393600,
				"2022-03-07": 410000,
				"2022-03-08": 426400,
				"2022-03-09": 442800,
				"2022-03-10": 459200,
				"2022-03-11": 475600,
				"2022-03-12": 492000,
				"2022-03-13": 508400,
				"2022-03-14": 524800,
				"2022-03-15": 541200,
				"2022-03-16": 557600,
				"2022-03-17": 574000,
				"2022-03-18": 590400,
				"2022-03-19": 606800,
				"2022-03-20": 623200,
				"2022-03-21": 639600,
				"2022-03-22": 656000,
				"2022-03-23": 672400,
				"2022-03-24": 688800,
				"2022-03-25": 705200,
				"2022-03-26": 721600,
				"2022-03-27": 738000,
				"2022-03-28": 754400,
				"2022-03-29": 770800,
				"2022-03-30": 787200,
				"2022-03-31": 803600,
				"2022-04-01": 820000,
				"2022-04-02": 836400,
				"2022-04-03": 852800,
				"2022-04-04": 869200,
				"2022-04-05": 885600,
				"2022-04-06": 902000,
				"2022-04-07": 918400,
				"2022-04-08": 934800,
				"2022-04-09": 951200,
				"2022-04-10": 967600,
				"2022-04-11": 984000,
				"2022-04-12": 1000400,
				"2022-04-13": 1016800,
				"2022-04-14": 1033200,
				"2022-04-15": 1049600,
				"2022-04-16": 1066000,
				"2022-04-17": 1082400,
				"2022-04-18": 1098800,
				"2022-04-19": 1115200
			}
		},
		"prev": {
			"hits": {
				"total": 0
			},
			"bandwidth": {
				"total": 0
			}
		}
	},
	"/v1/proxy/wp-plugins/stats?period=day": {
		"date": "2022-04-21",
		"hits": {
			"total": {
				"total": 68
			},
			"dates": {
				"2022-04-19": 68
			}
		},
		"bandwidth": {
			"total": {
				"total": 1115200
			},
			"dates": {
				"2022-04-19": 1115200
			}
		},
		"prev": {
			"hits": {
				"total": 67
			},
			"bandwidth": {
				"total": 1098800
			}
		}
	},
	"/v1/proxy/wp-plugins/stats?period=month": {
		"date": "2022-04-21",
		"hits": {
			"total": {
				"total": 1605
			},
			"dates": {
				"2022-03-21": 39,
				"2022-03-22": 40,
				"2022-03-23": 41,
				"2022-03-24": 42,
				"2022-03-25": 43,
				"2022-03-26": 44,
				"2022-03-27": 45,
				"2022-03-28": 46,
				"2022-03-29": 47,
				"2022-03-30": 48,
				"2022-03-31": 49,
				"2022-04-01": 50,
				"2022-04-02": 51,
				"2022-04-03": 52,
				"2022-04-04": 53,
				"2022-04-05": 54,
				"2022-04-06": 55,
				"2022-04-07": 56,
				"2022-04-08": 57,
				"2022-04-09": 58,
				"2022-04-10": 59,
				"2022-04-11": 60,
				"2022-04-12": 61,
				"2022-04-13": 62,
				"2022-04-14": 63,
				"2022-04-15": 64,
				"2022-04-16": 65,
				"2022-04-17": 66,
				"2022-04-18": 67,
				"2022-04-19": 68
			}
		},
		"bandwidth": {
			"total": {
				"total": 26322000
			},
			"dates": {
				"2022-03-21": 639600,
				"2022-03-22": 656000,
				"2022-03-23": 672400,
				"2022-03-24": 688800,
				"2022-03-25": 705200,
				"2022-03-26": 721600,
				"2022-03-27": 738000,
				"2022-03-28": 754400,
				"2022-03-29": 770800,
				"2022-03-30": 787200,
				"2022-03-31": 803600,
				"2022-04-01": 820000,
				"2022-04-02": 836400,
				"2022-04-03": 852800,
				"2022-04-04": 869200,
				"2022-04-05": 885600,
				"2022-04-06": 902000,
				"2022-04-07": 918400,
				"2022-04-08": 934800,
				"2022-04-09": 951200,
				"2022-04-10": 967600,
				"2022-04-11": 984000,
				"2022-04-12": 1000400,
				"2022-04-13": 1016800,
				"2022-04-14": 1033200,
				"2022-04-15": 1049600,
				"2022-04-16": 1066000,
				"2022-04-17": 1082400,
				"2022-04-18": 1098800,
				"2022-04-19": 1115200
			}
		},
		"prev": {
			"hits": {
				"total": 705
			},
			"bandwidth": {
				"total": 11562000
			}
		}
	},
	"/v1/proxy/wp-plugins/stats?period=week": {
		"date": "2022-04-21",
		"hits": {
			"total": {
				"total": 455
			},
			"dates": {
				"2022-04-13": 62,
				"2022-04-14": 63,
				"2022-04-15": 64,
				"2022-04-16": 65,
				"2022-04-17": 66,
				"2022-04-18": 67,
				"2022-04-19": 68
			}
		},
		"bandwidth": {
			"total": {
				"total": 7462000
			},
			"dates": {
				"2022-04-13": 1016800,
				"2022-04-14": 1033200,
				"2022-04-15": 1049600,
				"2022-04-16": 1066000,
				"2022-04-17": 1082400,
				"2022-04-18": 1098800,
				"2022-04-19": 1115200
			}
		},
		"prev": {
			"hits": {
				"total": 406
			},
			"bandwidth": {
				"total": 6658400
			}
		}
	},
	"/v1/proxy/wp-plugins/stats?period=year": {
		"date": "2022-04-21",
		"hits": {
			"total": {
				"total": 2346
			},
			"dates": {
				"2021-04-20": 0,
				"2021-04-21": 0,
				"2021-04-22": 0,
				"2021-04-23": 0,
				"2021-04-24": 0,
				"2021-04-25": 0,
				"2021-04-26": 0,
				"2021-04-27": 0,
				"2021-04-28": 0,
				"2021-04-29": 0,
				"2021-04-30": 0,
				"2021-05-01": 0,
				"2021-05-02": 0,
				"2021-05-03": 0,
				"2021-05-04": 0,
				"2021-05-05": 0,
				"2021-05-06": 0,
				"2021-05-07": 0,
				"2021-05-08": 0,
				"2021-05-09": 0,
				"2021-05-10": 0,
				"2021-05-11": 0,
				"2021-05-12": 0,
				"2021-05-13": 0,
				"2021-05-14": 0,
				"2021-05-15": 0,
				"2021-05-16": 0,
				"2021-05-17": 0,
				"2021-05-18": 0,
				"2021-05-19": 0,
				"2021-05-20": 0,
				"2021-05-21": 0,
				"2021-05-22": 0,
				"2021-05-23": 0,
				"2021-05-24": 0,
				"2021-05-25": 0,
				"2021-05-26": 0,
				"2021-05-27": 0,
				"2021-05-28": 0,
				"2021-05-29": 0,
				"2021-05-30": 0,
				"2021-05-31": 0,
				"2021-06-01": 0,
				"2021-06-02": 0,
				"2021-06-03": 0,
				"2021-06-04": 0,
				"2021-06-05": 0,
				"2021-06-06": 0,
				"2021-06-07": 0,
				"2021-06-08": 0,
				"2021-06-09": 0,
				"2021-06-10": 0,
				"2021-06-11": 0,
				"2021-06-12": 0,
				"2021-06-13": 0,
				"2021-06-14": 0,
				"2021-06-15": 0,
				"2021-06-16": 0,
				"2021-06-17": 0,
				"2021-06-18": 0,
				"2021-06-19": 0,
				"2021-06-20": 0,
				"2021-06-21": 0,
				"2021-06-22": 0,
				"2021-06-23": 0,
				"2021-06-24": 0,
				"2021-06-25": 0,
				"2021-06-26": 0,
				"2021-06-27": 0,
				"2021-06-28": 0,
				"2021-06-29": 0,
				"2021-06-30": 0,
				"2021-07-01": 0,
				"2021-07-02": 0,
				"2021-07-03": 0,
				"2021-07-04": 0,
				"2021-07-05": 0,
				"2021-07-06": 0,
				"2021-07-07": 0,
				"2021-07-08": 0,
				"2021-07-09": 0,
				"2021-07-10": 0,
				"2021-07-11": 0,
				"2021-07-12": 0,
				"2021-07-13": 0,
				"2021-07-14": 0,
				"2021-07-15": 0,
				"2021-07-16": 0,
				"2021-07-17": 0,
				"2021-07-18": 0,
				"2021-07-19": 0,
				"2021-07-20": 0,
				"2021-07-21": 0,
				"2021-07-22": 0,
				"2021-07-23": 0,
				"2021-07-24": 0,
				"2021-07-25": 0,
				"2021-07-26": 0,
				"2021-07-27": 0,
				"2021-07-28": 0,
				"2021-07-29": 0,
				"2021-07-30": 0,
				"2021-07-31": 0,
				"2021-08-01": 0,
				"2021-08-02": 0,
				"2021-08-03": 0,
				"2021-08-04": 0,
				"2021-08-05": 0,
				"2021-08-06": 0,
				"2021-08-07": 0,
				"2021-08-08": 0,
				"2021-08-09": 0,
				"2021-08-10": 0,
				"2021-08-11": 0,
				"2021-08-12": 0,
				"2021-08-13": 0,
				"2021-08-14": 0,
				"2021-08-15": 0,
				"2021-08-16": 0,
				"2021-08-17": 0,
				"2021-08-18": 0,
				"2021-08-19": 0,
				"2021-08-20": 0,
				"2021-08-21": 0,
				"2021-08-22": 0,
				"2021-08-23": 0,
				"2021-08-24": 0,
				"2021-08-25": 0,
				"2021-08-26": 0,
				"2021-08-27": 0,
				"2021-08-28": 0,
				"2021-08-29": 0,
				"2021-08-30": 0,
				"2021-08-31": 0,
				"2021-09-01": 0,
				"2021-09-02": 0,
				"2021-09-03": 0,
				"2021-09-04": 0,
				"2021-09-05": 0,
				"2021-09-06": 0,
				"2021-09-07": 0,
				"2021-09-08": 0,
				"2021-09-09": 0,
				"2021-09-10": 0,
				"2021-09-11": 0,
				"2021-09-12": 0,
				"2021-09-13": 0,
				"2021-09-14": 0,
				"2021-09-15": 0,
				"2021-09-16": 0,
				"2021-09-17": 0,
				"2021-09-18": 0,
				"2021-09-19": 0,
				"2021-09-20": 0,
				"2021-09-21": 0,
				"2021-09-22": 0,
				"2021-09-23": 0,
				"2021-09-24": 0,
				"2021-09-25": 0,
				"2021-09-26": 0,
				"2021-09-27": 0,
				"2021-09-28": 0,
				"2021-09-29": 0,
				"2021-09-30": 0,
				"2021-10-01": 0,
				"2021-10-02": 0,
				"2021-10-03": 0,
				"2021-10-04": 0,
				"2021-10-05": 0,
				"2021-10-06": 0,
				"2021-10-07": 0,
				"2021-10-08": 0,
				"2021-10-09": 0,
				"2021-10-10": 0,
				"2021-10-11": 0,
				"2021-10-12": 0,
				"2021-10-13": 0,
				"2021-10-14": 0,
				"2021-10-15": 0,
				"2021-10-16": 0,
				"2021-10-17": 0,
				"2021-10-18": 0,
				"2021-10-19": 0,
				"2021-10-20": 0,
				"2021-10-21": 0,
				"2021-10-22": 0,
				"2021-10-23": 0,
				"2021-10-24": 0,
				"2021-10-25": 0,
				"2021-10-26": 0,
				"2021-10-27": 0,
				"2021-10-28": 0,
				"2021-10-29": 0,
				"2021-10-30": 0,
				"2021-10-31": 0,
				"2021-11-01": 0,
				"2021-11-02": 0,
				"2021-11-03": 0,
				"2021-11-04": 0,
				"2021-11-05": 0,
				"2021-11-06": 0,
				"2021-11-07": 0,
				"2021-11-08": 0,
				"2021-11-09": 0,
				"2021-11-10": 0,
				"2021-11-11": 0,
				"2021-11-12": 0,
				"2021-11-13": 0,
				"2021-11-14": 0,
				"2021-11-15": 0,
				"2021-11-16": 0,
				"2021-11-17": 0,
				"2021-11-18": 0,
				"2021-11-19": 0,
				"2021-11-20": 0,
				"2021-11-21": 0,
				"2021-11-22": 0,
				"2021-11-23": 0,
				"2021-11-24": 0,
				"2021-11-25": 0,
				"2021-11-26": 0,
				"2021-11-27": 0,
				"2021-11-28": 0,
				"2021-11-29": 0,
				"2021-11-30": 0,
				"2021-12-01": 0,
				"2021-12-02": 0,
				"2021-12-03": 0,
				"2021-12-04": 0,
				"2021-12-05": 0,
				"2021-12-06": 0,
				"2021-12-07": 0,
				"2021-12-08": 0,
				"2021-12-09": 0,
				"2021-12-10": 0,
				"2021-12-11": 0,
				"2021-12-12": 0,
				"2021-12-13": 0,
				"2021-12-14": 0,
				"2021-12-15": 0,
				"2021-12-16": 0,
				"2021-12-17": 0,
				"2021-12-18": 0,
				"2021-12-19": 0,
				"2021-12-20": 0,
				"2021-12-21": 0,
				"2021-12-22": 0,
				"2021-12-23": 0,
				"2021-12-24": 0,
				"2021-12-25": 0,
				"2021-12-26": 0,
				"2021-12-27": 0,
				"2021-12-28": 0,
				"2021-12-29": 0,
				"2021-12-30": 0,
				"2021-12-31": 0,
				"2022-01-01": 0,
				"2022-01-02": 0,
				"2022-01-03": 0,
				"2022-01-04": 0,
				"2022-01-05": 0,
				"2022-01-06": 0,
				"2022-01-07": 0,
				"2022-01-08": 0,
				"2022-01-09": 0,
				"2022-01-10": 0,
				"2022-01-11": 0,
				"2022-01-12": 0,
				"2022-01-13": 0,
				"2022-01-14": 0,
				"2022-01-15": 0,
				"2022-01-16": 0,
				"2022-01-17": 0,
				"2022-01-18": 0,
				"2022-01-19": 0,
				"2022-01-20": 0,
				"2022-01-21": 0,
				"2022-01-22": 0,
				"2022-01-23": 0,
				"2022-01-24": 0,
				"2022-01-25": 0,
				"2022-01-26": 0,
				"2022-01-27": 0,
				"2022-01-28": 0,
				"2022-01-29": 0,
				"2022-01-30": 0,
				"2022-01-31": 0,
				"2022-02-01": 0,
				"2022-02-02": 0,
				"2022-02-03": 0,
				"2022-02-04": 0,
				"2022-02-05": 0,
				"2022-02-06": 0,
				"2022-02-07": 0,
				"2022-02-08": 0,
				"2022-02-09": 0,
				"2022-02-10": 0,
				"2022-02-11": 1,
				"2022-02-12": 2,
				"2022-02-13": 3,
				"2022-02-14": 4,
				"2022-02-15": 5,
				"2022-02-16": 6,
				"2022-02-17": 7,
				"2022-02-18": 8,
				"2022-02-19": 9,
				"2022-02-20": 10,
				"2022-02-21": 11,
				"2022-02-22": 12,
				"2022-02-23": 13,
				"2022-02-24": 14,
				"2022-02-25": 15,
				"2022-02-26": 16,
				"2022-02-27": 17,
				"2022-02-28": 18,
				"2022-03-01": 19,
				"2022-03-02": 20,
				"2022-03-03": 21,
				"2022-03-04": 22,
				"2022-03-05": 23,
				"2022-03-06": 24,
				"2022-03-07": 25,
				"2022-03-08": 26,
				"2022-03-09": 27,
				"2022-03-10": 28,
				"2022-03-11": 29,
				"2022-03-12": 30,
				"2022-03-13": 31,
				"2022-03-14": 32,
				"2022-03-15": 33,
				"2022-03-16": 34,
				"2022-03-17": 35,
				"2022-03-18": 36,
				"2022-03-19": 37,
				"2022-03-20": 38,
				"2022-03-21": 39,
				"2022-03-22": 40,
				"2022-03-23": 41,
				"2022-03-24": 42,
				"2022-03-25": 43,
				"2022-03-26": 44,
				"2022-03-27": 45,
				"2022-03-28": 46,
				"2022-03-29": 47,
				"2022-03-30": 48,
				"2022-03-31": 49,
				"2022-04-01": 50,
				"2022-04-02": 51,
				"2022-04-03": 52,
				"2022-04-04": 53,
				"2022-04-05": 54,
				"2022-04-06": 55,
				"2022-04-07": 56,
				"2022-04-08": 57,
				"2022-04-09": 58,
				"2022-04-10": 59,
				"2022-04-11": 60,
				"2022-04-12": 61,
				"2022-04-13": 62,
				"2022-04-14": 63,
				"2022-04-15": 64,
				"2022-04-16": 65,
				"2022-04-17": 66,
				"2022-04-18": 67,
				"2022-04-19": 68
			}
		},
		"bandwidth": {
			"total": {
				"total": 38474400
			},
			"dates": {
				"2021-04-20": 0,
				"2021-04-21": 0,
				"2021-04-22": 0,
				"2021-04-23": 0,
				"2021-04-24": 0,
				"2021-04-25": 0,
				"2021-04-26": 0,
				"2021-04-27": 0,
				"2021-04-28": 0,
				"2021-04-29": 0,
				"2021-04-30": 0,
				"2021-05-01": 0,
				"2021-05-02": 0,
				"2021-05-03": 0,
				"2021-05-04": 0,
				"2021-05-05": 0,
				"2021-05-06": 0,
				"2021-05-07": 0,
				"2021-05-08": 0,
				"2021-05-09": 0,
				"2021-05-10": 0,
				"2021-05-11": 0,
				"2021-05-12": 0,
				"2021-05-13": 0,
				"2021-05-14": 0,
				"2021-05-15": 0,
				"2021-05-16": 0,
				"2021-05-17": 0,
				"2021-05-18": 0,
				"2021-05-19": 0,
				"2021-05-20": 0,
				"2021-05-21": 0,
				"2021-05-22": 0,
				"2021-05-23": 0,
				"2021-05-24": 0,
				"2021-05-25": 0,
				"2021-05-26": 0,
				"2021-05-27": 0,
				"2021-05-28": 0,
				"2021-05-29": 0,
				"2021-05-30": 0,
				"2021-05-31": 0,
				"2021-06-01": 0,
				"2021-06-02": 0,
				"2021-06-03": 0,
				"2021-06-04": 0,
				"2021-06-05": 0,
				"2021-06-06": 0,
				"2021-06-07": 0,
				"2021-06-08": 0,
				"2021-06-09": 0,
				"2021-06-10": 0,
				"2021-06-11": 0,
				"2021-06-12": 0,
				"2021-06-13": 0,
				"2021-06-14": 0,
				"2021-06-15": 0,
				"2021-06-16": 0,
				"2021-06-17": 0,
				"2021-06-18": 0,
				"2021-06-19": 0,
				"2021-06-20": 0,
				"2021-06-21": 0,
				"2021-06-22": 0,
				"2021-06-23": 0,
				"2021-06-24": 0,
				"2021-06-25": 0,
				"2021-06-26": 0,
				"2021-06-27": 0,
				"2021-06-28": 0,
				"2021-06-29": 0,
				"2021-06-30": 0,
				"2021-07-01": 0,
				"2021-07-02": 0,
				"2021-07-03": 0,
				"2021-07-04": 0,
				"2021-07-05": 0,
				"2021-07-06": 0,
				"2021-07-07": 0,
				"2021-07-08": 0,
				"2021-07-09": 0,
				"2021-07-10": 0,
				"2021-07-11": 0,
				"2021-07-12": 0,
				"2021-07-13": 0,
				"2021-07-14": 0,
				"2021-07-15": 0,
				"2021-07-16": 0,
				"2021-07-17": 0,
				"2021-07-18": 0,
				"2021-07-19": 0,
				"2021-07-20": 0,
				"2021-07-21": 0,
				"2021-07-22": 0,
				"2021-07-23": 0,
				"2021-07-24": 0,
				"2021-07-25": 0,
				"2021-07-26": 0,
				"2021-07-27": 0,
				"2021-07-28": 0,
				"2021-07-29": 0,
				"2021-07-30": 0,
				"2021-07-31": 0,
				"2021-08-01": 0,
				"2021-08-02": 0,
				"2021-08-03": 0,
				"2021-08-04": 0,
				"2021-08-05": 0,
				"2021-08-06": 0,
				"2021-08-07": 0,
				"2021-08-08": 0,
				"2021-08-09": 0,
				"2021-08-10": 0,
				"2021-08-11": 0,
				"2021-08-12": 0,
				"2021-08-13": 0,
				"2021-08-14": 0,
				"2021-08-15": 0,
				"2021-08-16": 0,
				"2021-08-17": 0,
				"2021-08-18": 0,
				"2021-08-19": 0,
				"2021-08-20": 0,
				"2021-08-21": 0,
				"2021-08-22": 0,
				"2021-08-23": 0,
				"2021-08-24": 0,
				"2021-08-25": 0,
				"2021-08-26": 0,
				"2021-08-27": 0,
				"2021-08-28": 0,
				"2021-08-29": 0,
				"2021-08-30": 0,
				"2021-08-31": 0,
				"2021-09-01": 0,
				"2021-09-02": 0,
				"2021-09-03": 0,
				"2021-09-04": 0,
				"2021-09-05": 0,
				"2021-09-06": 0,
				"2021-09-07": 0,
				"2021-09-08": 0,
				"2021-09-09": 0,
				"2021-09-10": 0,
				"2021-09-11": 0,
				"2021-09-12": 0,
				"2021-09-13": 0,
				"2021-09-14": 0,
				"2021-09-15": 0,
				"2021-09-16": 0,
				"2021-09-17": 0,
				"2021-09-18": 0,
				"2021-09-19": 0,
				"2021-09-20": 0,
				"2021-09-21": 0,
				"2021-09-22": 0,
				"2021-09-23": 0,
				"2021-09-24": 0,
				"2021-09-25": 0,
				"2021-09-26": 0,
				"2021-09-27": 0,
				"2021-09-28": 0,
				"2021-09-29": 0,
				"2021-09-30": 0,
				"2021-10-01": 0,
				"2021-10-02": 0,
				"2021-10-03": 0,
				"2021-10-04": 0,
				"2021-10-05": 0,
				"2021-10-06": 0,
				"2021-10-07": 0,
				"2021-10-08": 0,
				"2021-10-09": 0,
				"2021-10-10": 0,
				"2021-10-11": 0,
				"2021-10-12": 0,
				"2021-10-13": 0,
				"2021-10-14": 0,
				"2021-10-15": 0,
				"2021-10-16": 0,
				"2021-10-17": 0,
				"2021-10-18": 0,
				"2021-10-19": 0,
				"2021-10-20": 0,
				"2021-10-21": 0,
				"2021-10-22": 0,
				"2021-10-23": 0,
				"2021-10-24": 0,
				"2021-10-25": 0,
				"2021-10-26": 0,
				"2021-10-27": 0,
				"2021-10-28": 0,
				"2021-10-29": 0,
				"2021-10-30": 0,
				"2021-10-31": 0,
				"2021-11-01": 0,
				"2021-11-02": 0,
				"2021-11-03": 0,
				"2021-11-04": 0,
				"2021-11-05": 0,
				"2021-11-06": 0,
				"2021-11-07": 0,
				"2021-11-08": 0,
				"2021-11-09": 0,
				"2021-11-10": 0,
				"2021-11-11": 0,
				"2021-11-12": 0,
				"2021-11-13": 0,
				"2021-11-14": 0,
				"2021-11-15": 0,
				"2021-11-16": 0,
				"2021-11-17": 0,
				"2021-11-18": 0,
				"2021-11-19": 0,
				"2021-11-20": 0,
				"2021-11-21": 0,
				"2021-11-22": 0,
				"2021-11-23": 0,
				"2021-11-24": 0,
				"2021-11-25": 0,
				"2021-11-26": 0,
				"2021-11-27": 0,
				"2021-11-28": 0,
				"2021-11-29": 0,
				"2021-11-30": 0,
				"2021-12-01": 0,
				"2021-12-02": 0,
				"2021-12-03": 0,
				"2021-12-04": 0,
				"2021-12-05": 0,
				"2021-12-06": 0,
				"2021-12-07": 0,
				"2021-12-08": 0,
				"2021-12-09": 0,
				"2021-12-10": 0,
				"2021-12-11": 0,
				"2021-12-12": 0,
				"2021-12-13": 0,
				"2021-12-14": 0,
				"2021-12-15": 0,
				"2021-12-16": 0,
				"2021-12-17": 0,
				"2021-12-18": 0,
				"2021-12-19": 0,
				"2021-12-20": 0,
				"2021-12-21": 0,
				"2021-12-22": 0,
				"2021-12-23": 0,
				"2021-12-24": 0,
				"2021-12-25": 0,
				"2021-12-26": 0,
				"2021-12-27": 0,
				"2021-12-28": 0,
				"2021-12-29": 0,
				"2021-12-30": 0,
				"2021-12-31": 0,
				"2022-01-01": 0,
				"2022-01-02": 0,
				"2022-01-03": 0,
				"2022-01-04": 0,
				"2022-01-05": 0,
				"2022-01-06": 0,
				"2022-01-07": 0,
				"2022-01-08": 0,
				"2022-01-09": 0,
				"2022-01-10": 0,
				"2022-01-11": 0,
				"2022-01-12": 0,
				"2022-01-13": 0,
				"2022-01-14": 0,
				"2022-01-15": 0,
				"2022-01-16": 0,
				"2022-01-17": 0,
				"2022-01-18": 0,
				"2022-01-19": 0,
				"2022-01-20": 0,
				"2022-01-21": 0,
				"2022-01-22": 0,
				"2022-01-23": 0,
				"2022-01-24": 0,
				"2022-01-25": 0,
				"2022-01-26": 0,
				"2022-01-27": 0,
				"2022-01-28": 0,
				"2022-01-29": 0,
				"2022-01-30": 0,
				"2022-01-31": 0,
				"2022-02-01": 0,
				"2022-02-02": 0,
				"2022-02-03": 0,
				"2022-02-04": 0,
				"2022-02-05": 0,
				"2022-02-06": 0,
				"2022-02-07": 0,
				"2022-02-08": 0,
				"2022-02-09": 0,
				"2022-02-10": 0,
				"2022-02-11": 16400,
				"2022-02-12": 32800,
				"2022-02-13": 49200,
				"2022-02-14": 65600,
				"2022-02-15": 82000,
				"2022-02-16": 98400,
				"2022-02-17": 114800,
				"2022-02-18": 131200,
				"2022-02-19": 147600,
				"2022-02-20": 164000,
				"2022-02-21": 180400,
				"2022-02-22": 196800,
				"2022-02-23": 213200,
				"2022-02-24": 229600,
				"2022-02-25": 246000,
				"2022-02-26": 262400,
				"2022-02-27": 278800,
				"2022-02-28": 295200,
				"2022-03-01": 311600,
				"2022-03-02": 328000,
				"2022-03-03": 344400,
				"2022-03-04": 360800,
				"2022-03-05": 377200,
				"2022-03-06": 393600,
				"2022-03-07": 410000,
				"2022-03-08": 426400,
				"2022-03-09": 442800,
				"2022-03-10": 459200,
				"2022-03-11": 475600,
				"2022-03-12": 492000,
				"2022-03-13": 508400,
				"2022-03-14": 524800,
				"2022-03-15": 541200,
				"2022-03-16": 557600,
				"2022-03-17": 574000,
				"2022-03-18": 590400,
				"2022-03-19": 606800,
				"2022-03-20": 623200,
				"2022-03-21": 639600,
				"2022-03-22": 656000,
				"2022-03-23": 672400,
				"2022-03-24": 688800,
				"2022-03-25": 705200,
				"2022-03-26": 721600,
				"2022-03-27": 738000,
				"2022-03-28": 754400,
				"2022-03-29": 770800,
				"2022-03-30": 787200,
				"2022-03-31": 803600,
				"2022-04-01": 820000,
				"2022-04-02": 836400,
				"2022-04-03": 852800,
				"2022-04-04": 869200,
				"2022-04-05": 885600,
				"2022-04-06": 902000,
				"2022-04-07": 918400,
				"2022-04-08": 934800,
				"2022-04-09": 951200,
				"2022-04-10": 967600,
				"2022-04-11": 984000,
				"2022-04-12": 1000400,
				"2022-04-13": 1016800,
				"2022-04-14": 1033200,
				"2022-04-15": 1049600,
				"2022-04-16": 1066000,
				"2022-04-17": 1082400,
				"2022-04-18": 1098800,
				"2022-04-19": 1115200
			}
		},
		"prev": {
			"hits": {
				"total": 0
			},
			"bandwidth": {
				"total": 0
			}
		}
	}
}
